ruby/build_jruby_release.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# This file should be executed with jruby
+
+set -ex
+
+# --- begin runfiles.bash initialization ---
+# Copy-pasted from Bazel's Bash runfiles library (tools/bash/runfiles/runfiles.bash).
+set -euo pipefail
+if [[ ! -d "${RUNFILES_DIR:-/dev/null}" && ! -f "${RUNFILES_MANIFEST_FILE:-/dev/null}" ]]; then
+  if [[ -f "$0.runfiles_manifest" ]]; then
+    export RUNFILES_MANIFEST_FILE="$0.runfiles_manifest"
+  elif [[ -f "$0.runfiles/MANIFEST" ]]; then
+    export RUNFILES_MANIFEST_FILE="$0.runfiles/MANIFEST"
+  elif [[ -f "$0.runfiles/bazel_tools/tools/bash/runfiles/runfiles.bash" ]]; then
+    export RUNFILES_DIR="$0.runfiles"
+  fi
+fi
+if [[ -f "${RUNFILES_DIR:-/dev/null}/bazel_tools/tools/bash/runfiles/runfiles.bash" ]]; then
+  source "${RUNFILES_DIR}/bazel_tools/tools/bash/runfiles/runfiles.bash"
+elif [[ -f "${RUNFILES_MANIFEST_FILE:-/dev/null}" ]]; then
+    source "$(grep -m1 "^bazel_tools/tools/bash/runfiles/runfiles.bash " \
+              "$RUNFILES_MANIFEST_FILE" | cut -d ' ' -f 2-)"
+else
+echo >&2 "ERROR: cannot find @bazel_tools//tools/bash/runfiles:runfiles.bash"
+exit 1
+fi
+# --- end runfiles.bash initialization ---
+
+# Make a temporary directory and move to it to do all packaging work
+mkdir -p tmp
```

Comment:
Since we're in the sandbox now do we need a tmp directory?

Reply:
I needed to make a new directory to start with a clean directory since this ends up being run in bazel-bin's runfile directory where a copy of all of the dependency files are.

Reply:
This gets run in bazel-bin/ruby/jruby_release.runfiles/com_google_protobuf/, which should *only* contain your dependencies.  Why do we need to copy all of them into a fresh folder?  This seems redundant, we could just run gem build from this directory